### ISSUE TITLE
Invalid Base64 encoding for AWS

### ISSUE THREAD

**Issue by egrieco** [quambene/pigeon-rs#2]
I'm testing `pigeon` with AWS and I keep receiving the following error:

```xml
  <Error>
    <Type>Sender</Type>
    <Code>MalformedInput</Code>
    <Message>Invalid Base64 encoding</Message>
  </Error>
```

Here is the command that I am running (actual email addresses hidden for privacy):

```zsh
pigeon send --connection aws --verbose --display sender@example.com receiver@example.com --message-file message.yaml
```

And here is the debug output:

```
Reading message file 'message.yaml' ...
Display message file: MessageTemplate {
    subject: "Pigeon Test",
    text: Some(
        "Testing pigeon for sending email.",
    ),
    html: Some(
        "Testing pigeon for sending email.",
    ),
}
Display email: Email {
    sender: "sender@example.com",
    receiver: "receiver@example.com",
    message: Message {
        subject: "Pigeon Test",
        text: Some(
            "Testing pigeon for sending email.",
        ),
        html: Some(
            "Testing pigeon for sending email.",
        ),
    },
    mime_format: From: sender@example.com
    To: receiver@example.com
    Subject: Pigeon Test
    MIME-Version: 1.0
    Date: Mon, 04 Jul 2022 18:21:37 +0000
    Content-Type: multipart/alternative;
     boundary="QtdoKChvSXDqIPYoKulOOZ8UFmHlV30mqzhHqI1G"

    --QtdoKChvSXDqIPYoKulOOZ8UFmHlV30mqzhHqI1G
    Content-Type: text/plain; charset=utf-8
    Content-Transfer-Encoding: 7bit

    Testing pigeon for sending email.
    --QtdoKChvSXDqIPYoKulOOZ8UFmHlV30mqzhHqI1G
    Content-Type: text/html; charset=utf-8
    Content-Transfer-Encoding: 7bit

    Testing pigeon for sending email.
    --QtdoKChvSXDqIPYoKulOOZ8UFmHlV30mqzhHqI1G--
    ,
}
```

It seems that the message is not being encoded as Base64.

I'll take another look at the source and see if I can further isolate or fix the error, time permitting.

**Comment by quambene** [comment]
Thanks for your bug report. Bug reproduced, having a look.

**Comment by egrieco** [comment]
I think I fixed it, but I'm having trouble with confirming that due to Amazon SES weirdness/BS.

Shall I put together a pull request anyway?

**Comment by quambene** [comment]
Thanks, I just pushed a fix to master and preparing a release for crates.io.

**Comment by egrieco** [comment]
Yep, just looked at commit ca9fbbc and I did the same thing. Seemed to fix the issue. 😄

**Comment by quambene** [comment]
Fixed in [v0.3.0](https://github.com/quambene/pigeon-rs/releases/tag/v0.3.0).
